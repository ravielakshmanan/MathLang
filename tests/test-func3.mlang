void printfunc(int a, float b, string c)
{
  print(a);
  printf(b);
  prints(c);
}

int main()
{
  printfunc(2,4.0,"MathLang");
  return 0;
}
