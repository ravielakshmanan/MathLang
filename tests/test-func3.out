2
4.000000
"MathLang"
